### PR TITLE
[MFA-334] feat: add read:mfa_policies and update:mfa_policies scopes

### PR DESCRIPTION
**Pull request by pmalouin**
## ✏️ Changes
  
The m2m client that is created by this extension should also include the new scopes `read:mfa_policies` and `update:mfa_policies`. They will be required by the latest version of auth0-deploy-cli (pending PR=https://github.com/auth0/auth0-deploy-cli/pull/252) that makes calls to endpoints requiring these scopes.
  
## 📷 Screenshots
 
🚫 
  
## 🔗 References
  
https://auth0team.atlassian.net/browse/MFA-334
  
## 🎯 Testing
  
🚫 This change has been tested in a Webtask
 
🚫 This change has unit test coverage
  
🚫 This change has integration test coverage
  
🚫 This change has been tested for performance
  
## 🚀 Deployment
  
✅ This can be deployed any time (the scopes are already supported by the management API).
  
## 🎡 Rollout
  
In order to verify that the deployment was successful we will install the extension on a tenant and see that the scopes are enabled for the client/application.
  
## 🖥 Appliance
  
**Note to reviewers:** ensure that this change is compatible with the Appliance.

### CODE REVIEW

File: webtask.json
Comment (pmalouin) [inline webtask.json:24]: Question for reviewer: will tenants that already installed this extension get the new scopes, or they need to uninstall+reinstall?

File: server/index.js
Comment (pmalouin) [inline server/index.js:52]: I'm basically blindly adding the scopes here, not sure I have to. Can the reviewer explain what these scopes here do and confirm whether I should add them here?

Comment (seejee) [reply server/index.js:52]: These scopes are eventually passed to the `/authorize` URL back on the RTA when the extension is launched. So, if the extension needs to make any Management API calls that require these scopes, the scopes should be added here. 👍

File: webtask.json
Comment (seejee) [reply webtask.json:24]: In this case, it will require an uninstall/reinstall so that the deploy CLI client is created with the new scopes. 

In other cases, if the changes are just code-related, then bumping the patch field of an extension will automatically roll out the new version of an extension to everyone who has it installed. Bumping the major or minor of an extension will require a manual uninstall/reinstall.

**Review by seejee — APPROVED** [review]

**Comment by seejee** [comment]
@pmalouin It's our policy now to require CMS approval for extensions. Once we have that, we'll get this merged and released.

Tracking ticket: https://auth0team.atlassian.net/browse/DXEX-672

File: server/index.js
Comment (pmalouin) [reply server/index.js:52]: Thanks for the review @seejee !

> So, if the extension needs to make any Management API calls that require these scopes, the scopes should be added here

In that case, they were probably unnecessary. The extension itself is not reading/updating the mfa policies, I think it's only concerned with clients and resource servers (we'd need to test it out to be sure). Although it's probably not critical that the scope is added here, it might have been better if the access token was granted only the minimal scopes necessary to perform its duty.

File: webtask.json
Comment (pmalouin) [reply webtask.json:24]: I tested this after the release of the new version. I was offered the option to "Update" my extension to the latest version via a button in the dashboard. After clicking that button, the new scopes were added to the API permissions granted to my M2M client, as expected 👍

Comment (seejee) [reply webtask.json:24]: Oh great! Thanks for testing that. I'll update our internal notes about this.

**Pull request merged**
